### ISSUE TITLE
NJS-512: invalid connection string parameters. NJS-514: syntax error in connection string

### ISSUE THREAD

**Issue by Shnitzelil** [oracle/node-oracledb#1664]
Version: 6.4.0

JS output:
oracledb.versionString=6.4.0
oracledb.oracleClientVersionString=undefined
process.platform=win32
process.version=v20.11.1
process.arch=x64
NJS-512: invalid connection string parameters.
NJS-514: syntax error in connection string

1. open command line (Windows)
2. (do not change working directory to node) point to nodejs full-path (that include brackets '(' and ')', like we have on Windows **Program Files (x86)**)
3. run any script with oracledb code.

Failure comes from parsing connection string like this
(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=myhostname.aws.domain.net)(PORT=1521))(CONNECT_DATA=(SERVICE_NAME=oracledb.domain.net)(CID=(PROGRAM=C:/Program Files (x86)/NodeJS/bin/node)(HOST=MyLaptop)(USER=kobi))))

maybe something like changing the utils.js
```javascript
this.program = process.argv0;  to 
this.program = process.argv0.split(/[\b]|\//).splice(-1).join('');
```

**Comment by sharadraju** [comment]
Hello,
Thanks for using node-oracledb.
The problem description is not clear.
Can you provide a sample script with the connect string that you are using.

**Comment by Shnitzelil** [comment]
```
const oracledb = require('oracledb');

console.log(`oracledb.versionString=${oracledb.versionString}`);    
console.log(`oracledb.oracleClientVersionString=${oracledb.oracleClientVersionString}`);       
console.log(`process.platform=${process.platform}`);
console.log(`process.version=${process.version}`);            	         
console.log(`process.arch=${process.arch}`);     

async function run() {
	let connection;
	try {
    	
        connection = await oracledb.getConnection({
                user          : "username",
                password      : "password",
	            privilege: oracledb.SYSDBA,
                connectString : "servername.aws.mydomain.net/orcl19.mydomain.net"
        });
        console.log(`Connected to ${connection.dbName}.${connection.dbDomain} - v.${connection.oracleServerVersionString}`);
        
	} catch (err) {
    	console.log(err.message);
	}
 
	if (connection) {
      try {
        await connection.close();    
      } catch (err) {
    	console.log(err.message);
      }
	}
}

run();
```

**Comment by sharadraju** [comment]
Thanks @Shnitzelil We will look into this.

**Comment by sharadraju** [comment]
This has been fixed in node-oracledb 6.5. @Shnitzelil Please verify.
